inst/tests/tests.Rraw

```diff
@@ -18784,6 +18784,11 @@ ans = c(
  "10:   83   64   41                 9                 9")
 # test where topn isn't necessary
 test(2264.8, print(DT, show.indices=TRUE), output=ans)
+# printing does not fail when indices are present
+test(2264.9, {
+  suppressWarnings( print(DT, show.indices=TRUE) )
```

Comment:
Instead of suppressing warnings, test that they don't occur. Try capturing the output from a `print(DT, show.indices=TRUE)` with the warning fixed and call `test(2264.9, print(DT, show.indices=TRUE), output = c(...example output goes here...))`.